report_mrp_bom_matrix/views/report_mrp_bom_matrix_view.xml

```diff
@@ -32,7 +32,7 @@
         <field name="name">report.mrp.bom.matrix.pivot</field>
         <field name="model">report.mrp.bom.matrix</field>
         <field name="arch" type="xml">
-            <pivot string="Report MRP BOM Matrix" create="0" delete="0">
```

Comment:
Why do you replace the create and delete options? The bom matrix is intended to be a report and we do not expect anyone to be able to create records.


![image](https://user-images.githubusercontent.com/7683926/36147849-1f4f3afc-10ba-11e8-9027-5ff56b0397fc.png)